### DISCUSSION TITLE
Why was there a number of tokens reduction for these chronos models compared to the t5 models?

### DISCUSSION THREAD

**Discussion by SimSef** [amazon-science/chronos-forecasting#124] [Q&A] [answered]
Hello there,
I would like to ask why was there the reduction to only 4096 params from the model it was built from?
And if i have the compute wouldnt I be better of using the original model chronos was based on, given the number of tokens?
However i am guessing it would just be an empty model right, but the pro would be i could use covariates perhaps?
Thanks for answering.

**Comment by abdulfatir** [comment, marked as answer]
@CoCoNuTeK Please follow the issue guidelines in the repo and use discussions for Q/A. Issues are intended for issues (such as bugs) in the code. 

The vocab size in the context of Chronos relates to the precision. 4096 was a reasonable choice. While larger values may improve precision, note that you don't want the bins to be too fine. In that case very few items may fall into those bins which may lead to the model not learning the distribution properly. Please check out the paper for discussion on such design choices: https://arxiv.org/abs/2403.07815
